rice-framework/krad-web-framework/src/main/java/org/kuali/rice/krad/web/filter/CharsetFilter.java

```diff
@@ -53,7 +53,6 @@ public void doFilter(ServletRequest request, ServletResponse response,
          * Set the default response content type and encoding
          */
         response.setContentType("text/html; charset=UTF-8");
-        response.setCharacterEncoding("UTF-8");

```

Comment:
What necessitated the need to remove the character encoding? I don't see that in the 2.5 change linked here.

Reply:
The setter appears to have been removed in the newer servlet spec.  It was removed due to compilation issues.

Reply:
hmm, i don't see where it was removed, at least not in servlet spec 4.0:

http://tomcat.apache.org/tomcat-9.0-doc/servletapi/javax/servlet/ServletResponse.html#setCharacterEncoding-java.lang.String-

That said, from the javadoc...

"Calling setContentType(java.lang.String) with the String of text/html and calling this method with the String of UTF-8 is equivalent with calling setContentType with the String of text/html; charset=UTF-8."

So it sounds like it's superfluous anyway given how we are invoking setContentType

Reply:
I found this was actually caused by duplicate servlet-api JARs on the classpath.  I fixed those up and reverted this change since it was no longer necessary.